src/purchase/utils.py

```diff
@@ -31,3 +39,59 @@ def store_leftover_paper_support(paper, purchase, leftover_amount):
         item=paper,
         hold_type=Escrow.AUTHOR_RSC,
     )
+
+
+def create_fundraise_with_escrow(
```

Comment:
This is not a comment requiring immediate action but I wanted to share a suggestion: For new apps, we should avoid putting business logic into utility packages (`utils.py`). For better cohesion (and testability), it would be preferable to organize business logic in service packages or classes as done in https://github.com/ResearchHub/researchhub-backend/blob/555c0feedd6ca52382f3d8bf605c4f9957d9c505/src/paper/services/storage_service.py#L14 Since `purchase` is an older app, we can continue as it.